### PR TITLE
Create alperenag.md

### PR DESCRIPTION
**Pull request by alperenag**

### CODE REVIEW

File: docs/contributing/contributors/alperenag.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+tags:
+  - contributors
+  - aerospace engineeer
```
Comment (owlot) [inline docs/contributing/contributors/alperenag.md:4]: Remove one `e` from `egineeer`

**Review by owlot — CHANGES_REQUESTED** [review]
Can you please add these words to the .cspell.project-words.txt file?
alperenag
Ahmet
Alperen
Gündoğan

If you're on a unix system, you can also run make cpsell-add-words to have the unknown words automatically added.

Make sure to add the file changes and commit afterwards 😃

**Comment by alperenag** [comment]
> Can you please add these words to the .cspell.project-words.txt file? alperenag Ahmet Alperen Gündoğan
> 
> If you're on a unix system, you can also run make cpsell-add-words to have the unknown words automatically added.
> 
> Make sure to add the file changes and commit afterwards 😃

Done, with the help of Owlot 😃

File: docs/contributing/contributors/alperenag.md (hunk 1 above)
Comment (alperenag) [reply docs/contributing/contributors/alperenag.md]: I think it's correct now?

**Review by owlot — APPROVED** [review]

**Pull request merged**